components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -4,6 +4,7 @@
  * See License-AGPL.txt in the project root for license information.
  */
 
+import { list as blocklist } from "the-big-username-blacklist";
```

Comment:
**praise**: Nice one! Thanks @corneliusludmann for also suggesting this in https://github.com/gitpod-io/gitpod/issues/4469#issuecomment-858657883. 🏀